docs/VISION.md

```diff
@@ -0,0 +1,23 @@
+# OpenTelemetry Service Long-term Vision
```

Comment:
one point that needs to be covered is a relationship with FluentBit and Prometheus. Maybe in a separate document.

Reply:
I am not familiar with this area, so perhaps someone else can add it separately (or I can do it later once I know more).

Reply:
Created issue for the future: https://github.com/open-telemetry/opentelemetry-service/issues/10